matlab/src/ObjectPrx.cpp

```diff
@@ -236,38 +236,16 @@ extern "C"
 
         try
         {
-            Ice::Context ctx;
-            getContext(context, ctx);
-            auto ok = restoreProxy(self)->ice_invoke(op, mode, params, v, ctx);
-            mxArray* results = 0;
-            if (!v.empty())
+            const Ice::Context* ctxPtr = &Ice::noExplicitContext;
+            Ice::Context cppContext;
+            if (!mxIsEmpty(context))
```

Comment:
context is now either an empty array or a configured dictionary. The configured dictionary may have no entry, but is never "empty".